### ISSUE TITLE
About the data involved in healthy.json

### ISSUE THREAD

**Issue by wrxjt** [MrGiovanni/SyntheticTumors#3]
This dataset requires manual download of four datasets, but the text does not specify whether Chaos uses 20 images from the training set or 20 images from the test set. For the work of the fllow author, I need to first reproduce the author's effect, can the author describe this dataset in detail? Or can you provide the dataset involved in this healthy.json setting? Thank you

**Comment by qixinhu11** [comment]
Thank you for showing interest in our work. The data we utilize are sourced from the CHAOS training set. 
The original CHAOS dataset includes .dcm files for the images and .png files for the segmentations. 
You can obtain the original data format by visiting "https://chaos.grand-challenge.org/Download/". 
Alternatively, you may choose to directly download the .nii.gz file through the following links: "wget https://www.dropbox.com/s/lzrhirei2t2vuwg/03_CHAOS.tar.gz" and "wget https://www.dropbox.com/s/8e3hlza16vor05s/label.zip".
